service/debugger/debugger.go

```diff
@@ -1518,6 +1518,16 @@ func (d *Debugger) Recorded() (recorded bool, tracedir string) {
 	return d.target.Recorded()
 }
 
+// ReturnValues returns the return values of the function we just stepped out of.
```

Comment:
No longer used?

Reply:
Reused for ReturnValues of a given thread.